tests/test_coordinates.py

```diff
@@ -84,9 +84,9 @@ def test_GDZ_in_kilometers(self):
         got = cc_km.convert('GDZ', 'sph')
         expected = cc_Re.convert('GDZ', 'sph')
         # same valued output in km regardless of units of input
-        np.testing.assert_approx_equal(expected.radi, got.radi, significant=6)
-        np.testing.assert_approx_equal(expected.lati, got.lati, significant=6)
-        np.testing.assert_approx_equal(expected.long, got.long, significant=6)
+        np.testing.assert_almost_equal(expected.radi, got.radi, decimal=6)
```

Comment:
Just to note that the numpy docs discourage use of this function, which might be cause to look into switching comparison functions across the whole test suite (I don't think we're using their recommended functions anywhere)

Reply:
Yep, they also discourage `assert_approx_equal` so in some sense it's the closest equivalent and this is the smallest workable change :) I think in most cases that we're using it's okay, but worth considering a full scrub in the future.